students/conferences.md

```diff
@@ -83,7 +83,7 @@ We are not able to support our students with travel and accommodation. If you ha
 
 Interested? Any further questions? Write us an <a href="mailto:contact@rgsoc.org">email</a> and we can talk about the details.
 
-<h3 id="conferences">Conference Sponsors in 2017</h3>
```

Comment:
since some of the conferences that supported our 2017 students have not actually happened yet, I thought it would be nice to leave their logos on for a while, until we kick off our program. We didn't add the conferences on the page until fairly late last year. 
Not sure what is the best way to coordinate this though. Cutting at the 2017/2018 mark could be confusing, since the upcoming 2018 conferences are supporting our 2017 students and not the 2018 ones. Do you have a suggestion on handling this? Or should we just forget it?

Reply:
Yeah, I wasn't sure about this one either. In my opinion, it makes sense to leave this on the website until we kickoff the new edition or even until we have a list of conferences for 2018 to replace it. I guess new participants will hardly mistake this for conferences available for them to participate 😂
Another option could be to remove the conferences that already happened and keep only the ones from 2018. But we wouldn't gain much from this and it's much nicer to keep their support public